server/routes/index.ts

```diff
@@ -574,6 +580,65 @@ export function defineRoutes(router: IRouter) {
     }
   );
 
+  router.get(
+    {
+      path: `${API_ENDPOINT_DASHBOARD_SIGNIN_OPTIONS}`,
+      validate: false,
+      options: {
+        authRequired: false,
+      },
+    },
+    async (
+      context,
+      request,
+      response
+    ): Promise<IOpenSearchDashboardsResponse<any | ResponseError>> => {
+      const client = context.security_plugin.esClient.asScoped(request);
+      let esResp;
+      try {
+        esResp = await client.callAsInternalUser('opensearch_security.dashboardsinfo');
+        return response.ok({
+          body: esResp.dashboard_signin_options,
+        });
+      } catch (error) {
+        return errorResponse(response, error);
+      }
+    }
+  );
+
+  router.put(
```

Comment:
in what cases will we need put call?

Reply:
We need to send `put` request every time an admin changes the dashboard sign in options and we need to update them in backend.

https://github.com/davidosorno/security-dashboards-plugin/blob/3905b6be757c5fbaf7552db1898a27b34d8b6f37/public/utils/dashboards-info-utils.tsx#L38